### DISCUSSION TITLE
Custom shortcut?

### DISCUSSION THREAD

**Discussion by enzosanchezariel** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#462] [Q&A]
Can I make a shortcut to launch the assistant? I don't use the tray icons

**Comment by ASaydam** [comment]
I own a logitech keyboard with custom keys (G1-G2- etc.), so I created a macro for win+shift+A keys and binded it to G6 key. So I can open Google Assistant with only G6 key. I know it is an external solution but you may want to try it if you have a similar keyboard or mouse until the app itself supports custom shortcut.

**Comment by im-coder-lg** [comment]
> I own a logitech keyboard with custom keys (G1-G2- etc.), so I created a macro for win+shift+A keys and binded it to G6 key. So I can open Google Assistant with only G6 key. I know it is an external solution but you may want to try it if you have a similar keyboard or mouse until the app itself supports custom shortcut.

That is a very good answer, however, for laptops, you need external software that can access root to change keybindings.
Maybe, if you want to change keybindings, you must wait for the feature OR get an open-source, secure keybinding changer.
